ci-operator/config/open-cluster-management/application-ui/open-cluster-management-application-ui-main.yaml

```diff
@@ -28,6 +28,42 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: e2e-cluster
+  steps:
+    dependencies:
+      COMPONENT_IMAGE_REF: application-ui
+    env:
+      CLUSTERPOOL_GROUP_NAME: AppLifecycle
+      CLUSTERPOOL_HOST_NAMESPACE: app
+      CLUSTERPOOL_HOST_PROW_KUBE_SECRET: ocm-app-ui-clusterpool
+      CLUSTERPOOL_LIST_INCLUSION_FILTER: prow
+    test:
+    - as: test
+      commands: |
+        export GITHUB_PRIVATE_URL=$(cat "/etc/e2e-secrets/github-private-url") \
+        export GITHUB_USER=$(cat "/etc/e2e-secrets/github-user") \
+        export GITHUB_TOKEN=$(cat "/etc/e2e-secrets/github-token") \
+        export OBJECTSTORE_PRIVATE_URL=$(cat "/etc/e2e-secrets/objectstore-private-url") \
+        export OBJECTSTORE_ACCESS_KEY=$(cat "/etc/e2e-secrets/objectstore-access-key") \
+        export OBJECTSTORE_SECRET_KEY=$(cat "/etc/e2e-secrets/objectstore-secret-key") \
+        export HELM_PRIVATE_URL=$(cat "/etc/e2e-secrets/helm-private-url") \
+        export HELM_USERNAME=$(cat "/etc/e2e-secrets/github-user") \
+        export HELM_PASSWORD=$(cat "/etc/e2e-secrets/github-token") \
+        export HELM_CHART_NAME=$(cat "/etc/e2e-secrets/helm-chart-name") \
+        export ANSIBLE_URL=$(cat "/etc/e2e-secrets/ansible-url") \
+        export ANSIBLE_TOKEN=$(cat "/etc/e2e-secrets/ansible-token") \
+        export SELF="make -f Makefile.prow" \
+        make -f Makefile.prow test-e2e
+      credentials:
+      - mount_path: /etc/e2e-secrets
+        name: e2e-secrets
+        namespace: test-credentials
+      from: src
```

Comment:
Let's run directly from the test image here.

Reply:
Thanks! Will make the change accordingly